dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/service/RegistryServerSyncTest.java

```diff
@@ -0,0 +1,86 @@
+package org.apache.dubbo.admin.service;
```

Comment:
Apache license announcement is required here.

Reply:
ok